api-server/server/boot/certificate.js

```diff
@@ -302,26 +306,27 @@ function createVerifyCert(certTypeIds, app) {
           [certType]: true
         };
 
-        if (challenge) {
-          const { id, tests, challengeType } = challenge;
-          if (
-            !user[certType] &&
-            !isCertified(tests, user.completedChallenges)
-          ) {
-            return Observable.just(notCertifiedMessage(certName));
-          }
-          updateData = {
-            ...updateData,
-            completedChallenges: [
-              ...user.completedChallenges,
-              {
-                id,
-                completedDate: new Date(),
-                challengeType
-              }
-            ]
-          };
+        // certificate doesn't exist or
+        // connection error
+        if (!challenge) {
```

Comment:
Given that this should not be happening, I think this should be reported to Sentry.  It might be end up being a bit spammy, but if this happens a lot we need to know.